src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -107,6 +107,7 @@ public function build(string $version): string
         $this->xmlConfigurationHelper->replaceWithAbsolutePaths($xPath);
         $this->xmlConfigurationHelper->setStopOnFailure($xPath);
         $this->xmlConfigurationHelper->deactivateColours($xPath);
+        $this->xmlConfigurationHelper->deactivateResultCaching($xPath);
         $this->xmlConfigurationHelper->removeCacheResultFile($dom, $xPath);
```

Comment:
The question is whether we still need to remove the `cacheResultFile` attribute, then.

What do you think?